src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-CSharp/Properties/launchSettings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "iisSettings": {
+//#if (WindowsAuth)
+    "windowsAuthentication": true,
+    "anonymousAuthentication": false,
+//#else
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+//#endif
+    "iisExpress": {
+      "applicationUrl": "http://localhost:55556/",
+//#if (RequiresHttps)
+      "sslPort": 43434
+//#else
+      "sslPort": 0
+//#endif
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+//#if (RequiresHttps)
+      "launchUrl": "https://localhost:43434/",
```

Comment:
Why does it launch at the root URL here, but at `api/values` if you don't require HTTPS?